.python-version

```diff
@@ -0,0 +1 @@
+3.9.19
```

Comment:
storey needs to support multiple python versions, not just this specific one. And I wouldn't want to have to keep bumping this for every patch release.

Reply:
You don't need to bump this on every release.
This file is for development purposes, for tools such as `pyenv`, `uv`, and GitHub Actions.

Reply:
Removed in 1ad4021